picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -670,6 +662,7 @@ public Builder newBuilder() {
     @Nullable private Listener listener;
     private final List<RequestTransformer> requestTransformers = new ArrayList<>();
     private final List<RequestHandler> requestHandlers = new ArrayList<>();
+    private final List<EventListener> eventListeners = new ArrayList<>();
```

Comment:
wasn't sure whether single vs multiple event listeners made sense.  opted for multiple.  can be convinced otherwise.